client/src/components/events/events.js

```diff
@@ -1,30 +1,27 @@
 import React from 'react' ;
-import {EventCard} from './eventsCardComponent'
 import HeaderC from '../header/index';
 import Navbar from '../navbar/index';
-import './eventsCardComponent.css' ;
-
+import './events.css';
+import EventCard from './eventsCardComponent';
 
 export default class Events extends React.Component {
 
 
   render() {
     return (
       <div>
+      <HeaderC/>
+    <h1>Events</h1>
+    <Navbar/>
+   <div className="twobuttons">
+   <button className="next">This Week:12-19 May</button>   <button>Next Week:19-26May </button>
```

Comment:
1- align them
2- ` <button>Next Week:19-26May </button>` why you have this as button ?
3- use the `StyledButton ` instead